packages/icons/glyphsMapping.json

```diff
@@ -928,5 +928,11 @@
   },
   "leaf": {
     "start": "\ue131"
+  },
+  "permissions": {
+    "start": "\ue133"
+  },
```

Comment:
Avoid naming icons having in mind where they are going to be used, instead give a literal name. 
Check the [naming guidelines here](https://zeroheight.com/2b98d65de/p/490006-iconography/t/246d00)

I would suggest changing the permissions name to `user-lock` or something similar